guides/common/modules/ref_customizing-ansible-configuration.adoc

```diff
@@ -7,8 +7,9 @@ However, you can customize other Ansible configuration options as usual.
 {Project} stores the essential Ansible configuration as environment variables in `/etc/foreman-proxy/ansible.env`.
 This file is managed by the {foreman-installer}.
 
-Ansible reads configuration from `/etc/ansible/ansible.cfg` and the environment provided by {SmartProxy}.
-If you need to customize Ansible configuration, you can do so in the `/etc/ansible/ansible.cfg` file.
+Ansible reads configuration from a configuration file and the environment provided by {SmartProxy}.
+If you need to customize Ansible configuration, you can do so in the system-wide `/etc/ansible/ansible.cfg` file or in the `/usr/share/foreman-proxy/.ansible.cfg` file in the home directory of the `foreman-proxy` user.
+Note that if you use `/usr/share/foreman-proxy/.ansible.cfg`, Ansible spawned by {Project} ignores configuration in `/etc/ansible/ansible.cfg`.
```

Comment:
I rewrote this bit. @adamruzicka Will this work?

Reply:
lgtm